src/main.h

```diff
@@ -38,7 +38,7 @@ gboolean main_is_realized(void);
 
 typedef struct
 {
-	gboolean	new_instance;
+	gint		new_instance;
```

Comment:
Yes I thought about it.  Can you suggest names for it?  Basically -1 = forced disabled, 0 = default and disabled (but can be changed to 1 by runtime), and 1 = explicitly enabled.